src/transaction.py

```diff
@@ -179,6 +179,26 @@ class Sell(Transaction):
     selling_value: Optional[decimal.Decimal] = None
 
 
+class MarginFee(Transaction):
+    """Fees for margin trading"""
+
+    pass
+
+
+class MarginGain(Transaction):
+    """Gains from margin trading.
+    This is already a taxable value, no buy/sell calculation required."""
+
+    pass
+
+
+class MarginLoss(Transaction):
+    """Losses from margin trading.
+    This is already a taxable value, no buy/sell calculation required."""
+
+    pass
```

Comment:
When possible, it might be nicer to have classes `MarginStart`, `MarginFee`, `MarginClose` instead.
`MarginClose` should have a `fees: list[MarginFee]` variable. The fees must be linked to the MarginClose for tax evaluation.